harness/src/lib.rs

```diff
@@ -177,8 +177,9 @@ impl Mollusk {
             .collect::<Vec<_>>();
 
         let transaction_accounts = [(self.program_id, self.program_account.clone())]
-            .into_iter()
+            .iter()
             .chain(accounts)
+            .cloned()
```

Comment:
Yeah, this was exactly what I had. Unfortunate, but I agree the API is better served with a reference.